src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/showRawYamlFalseInherited.groovy

```diff
@@ -0,0 +1,34 @@
+podTemplate(showRawYaml: false) { podTemplate(yaml: """
```

Comment:
BTW prefer `'''` to `"""` (and `'` to `"`) whenever you are not actually using Groovy interpolation.